test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTrimTest.java

```diff
@@ -100,4 +100,54 @@ public void testCheckpointTrimDuringPlayback() throws Exception {
         assertThat(newTestMap)
                  .containsKeys("a", "b", "c");
     }
+
+    @Test
+    public void testCheckpointTrimDuringPlayback2() throws Exception {
```

Comment:
Also, rename this test to what it actually tests now (transactionalReadAfterCheckpoint would work).